src/topology/algebra/group.lean

```diff
@@ -67,10 +67,18 @@ by { ext, refl }
 lemma is_open_map_mul_left (a : G) : is_open_map (λ x, a * x) :=
 (homeomorph.mul_left a).is_open_map
 
+@[to_additive is_open.left_add_coset]
+lemma is_open.left_coset {U : set G} (x : G) (h : is_open U) : is_open (left_coset x U) :=
```

Comment:
I think you need to swap the order of the arguments for dot notation to work:
```suggestion
lemma is_open.left_coset {U : set G} (h : is_open U) (x : G) : is_open (left_coset x U) :=
```
and likewise with the other lemmas below

Reply:
Dot notation doesn't care about the argument order here, the original was fine

Reply:
I think it does matter in some situations. For example the following:
```lean
example {U : set G} (h : is_open U) : ∀ x, is_open (left_coset x U) := h.left_coset
```
only works when `is_open.left_coset` is defined with `x` following `h`.

I admit that the ordering does not matter for something of the form:
```lean
example {U : set G} (h : is_open U) (x : G) : is_open (left_coset x U) := h.left_coset x
```
which is likely to be the more typical usage.

I'm happy to be corrected if there's more to it!